lib/paragraphs.rb

```diff
@@ -5,19 +5,23 @@ class Para
 
   def paragraphs(str_arr)
     str_arr.each_with_index.map do |str, idx|
-      if str.include?("\n") == true || str.include?("<h") == true || str.include?("<li>")
+      prev_line = prev_line(str_arr, idx, 1)
+      next_line = next_line(str_arr, idx, 1)
+      prev_sec_line = prev_line(str_arr, idx, 1)
+      str = case
```

Comment:
It's weird, when I wrote it that way it wouldn't take the conditionals it would go straight to the else statement